zh_cn.lang

```diff
@@ -299,8 +284,10 @@ gui.xaero_sort_name=名称
 gui.xaero_sort_symbol=首字母
 gui.xaero_sort_distance=距离
 gui.xaero_box_cave_maps=一个可以显示地面之下的地图模式，主要用来显示地底的洞穴和建筑内部。\n 该参数指的是在你的头顶上的“正方形”范围内有固实方块遮挡时启用洞穴模式。
+gui.xaero_box_cave_maps2=一个可以显示地面之下的地图模式，主要用来显示地底的洞穴和建筑内部。\n 该参数指的是在你的头顶上的“正方形”范围内有固实方块遮挡时启用洞穴模式。固实方块不要求在同一个 Y 轴高度。
 gui.xaero_box_entity_amount=指定每个实体类别的容量极限。
 gui.xaero_box_distance=指定显示哪一个游戏内路径点的距离。
+gui.xaero_box_distance=指定显示哪一个游戏内路径点的距离。这并不保证有多个路径点时它们的距离都会被显示，对于这个需求，请启用“多路径点距离显示”选项。
```

Comment:
It's supposed to be _gui.xaero_box_distance2_.
And the translation back to English seems wrong here too. 
"This does not guarantee that when there are multiple waypoints, their distances will be displayed, for this requirement, enable the "Multi-waypoint distance display" option."

Is not the same as this, at all:
"This does not guarantee that it will be shown for multiple waypoints at once. Check out the "Multiple Waypoint Distances" setting as well."

Please check that it actually basically means "only 1 distance is shown at a time, even if multiple waypoints are close enough to the crosshair, unless configured otherwise".